remove python refernce from the readme and the main py file (#2)
pbstck/homework: README.md

### Work

> Your job is to implement the funtion `merge` in whatever language you want. The pseudo-code function prototype is as follow:

Reviewer: ```suggestion
Your job is to implement the `merge` function. You can choose any language. The pseudo-code function prototype is as follow:
```
